README.md

```diff
@@ -27,6 +27,23 @@ The MT-GenEval benchmark was released in the EMNLP 2022 paper [MT-GenEval: A Cou
 The data is originally sourced from Wikipedia. 
 We include sentence-level development and test segments in `data/sentences/` and inter-sentence test segments in `data/context/`. 
 
+## Compute accuracy
```

Comment:
Thanks for updating the README!

For ease of use, let's set it up so that evaluations can be run directly from the command line by passing in arguments like path to system translations, target language, and subset (contextual/counterfactual). Ideally users should not have to modify the code itself (particularly since there are large modifications for contextual vs. counterfactual). For example:

./accuracy_metric.py --hypotheses [PATH_FOR_YOUR_SYSTEM_TRANSLATIONS] --target ru --subset contextual